pkg/replication/README.md

```diff
@@ -1,44 +0,0 @@
-# Replication: facilities for scheduling a pod across multiple nodes
```

Comment:
Why kill the README? It's nice to have per-package docs, or if you are replacing with a wiki page at least link to it.